tests/debugpy/server/test_cli.py

```diff
@@ -62,16 +72,15 @@ def parse(args):
     return parse
 
 
+# Test a combination of command line switches
 @pytest.mark.parametrize("target_kind", ["file", "module", "code"])
 @pytest.mark.parametrize("mode", ["listen", "connect"])
 @pytest.mark.parametrize("address", ["8888", "localhost:8888"])
 @pytest.mark.parametrize("wait_for_client", ["", "wait_for_client"])
-@pytest.mark.parametrize("script_args", ["", "script_args"])
```

Comment:
Is this test covered by the new unit tests?

Reply:
No, and I think I need to change my logic anyway. I didn't add a test for this case, I will give it a shot